### PR TITLE
build(release): docs commits should trigger minor release

### PR DESCRIPTION
**Pull request by ruxandrafed**
## Overview

Confirmed that this works (still extending centralized config, but adding a custom rule so that `docs` commits trigger minor releases. The default settings are for `docs` commits to not trigger a release, but it doesn't make sense for documentation repos such as this.

I also added more details in the README and in the CONTRIBUTING guide re. commit message conventions to follow.

---

#### Meta

> Please read and confirm each of the following:

- [ ] this topic was discussed in the [Technology Forum][technology-forum] _(ignore if the pull request represents small changes)_
- [x] provided a descriptive topic and overview of contribution
- [x] documentation format follows the [topic template][template]
- [x] fork is up to date _(Hint: ["Syncing a Fork"][guide-forks])_
- [x] "work in progress" commits are squashed _(Hint: ["Squashing Commits"][guide-squash])_
- [x] commits follow the [Conventional ChangeLog][conventional-changelog] format
- [x] no sensitive content included, such as:
  - content considered competitive intelligence
  - security & privacy policy violating content
  - keys, tokens or credentials

[template]: https://github.com/telus/reference-architecture/blob/master/.template.md
[conventional-changelog]: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular
[guide-forks]: https://help.github.com/articles/syncing-a-fork/
[guide-squash]: https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History
[technology-forum]: https://github.com/telus/technology-forum]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,9 @@
     "setup-local": "npx install-group peer --package @telus/build-essential --no-save"
   },
   "release": {
-    "extends" : "@telus/semantic-release-config/github"
+    "extends": "@telus/semantic-release-config/github",
+    "releaseRules": [
```
Comment (ahmadnassri) [inline package.json]: why is this repo not using the standard `.releaserc` file like the other ones?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ You should usually open a pull request in the following situations:
 
 A pull request doesn't have to represent finished work. It's usually better to open a pull request early on, so others can watch or give feedback on your progress. Just mark it as a "WIP" (Work in Progress) in the subject line. You can always add more commits later.
 
-As your write commits, be sure to check the [pull request template](./PULL_REQUEST_TEMPLATE.md) for guidance.
+As your write commits, be sure to check the [pull request template](./PULL_REQUEST_TEMPLATE.md) for guidance. Your commits should follow the [karma format][karma-format].
```
Comment (kspaans) [inline .github/CONTRIBUTING.md]: Should be "as **you** write commits"

File: package.json (hunk 1 above)
Comment (kspaans) [reply package.json]: Seems reasonable in this case for a repo that's strictly documentation.

Comment (ruxandrafed) [reply package.json]: @ahmadnassri I guess it was easier as I was testing. I don't mind leaving it here, but if you feel strong about this .. :D

Comment (ruxandrafed) [reply package.json]: done! moved 😄

**Pull request merged**